.github/workflows/ci.yaml

```diff
@@ -363,6 +363,7 @@ jobs:
           git submodule update --init --recursive --jobs 7
           ./project/scripts/sbt "community-build/testOnly dotty.communitybuild.CommunityBuildTestC"
 
+<<<<<<< HEAD
```

Comment:
You have some unresolved merge conflicts

Reply:
Locally I don't have those conflicts. Fetching / pulling doesn't change anything. Am I going insane?

Reply:
I'll check that on the fresh repo clone.

Reply:
Should be better now.